### PR TITLE
[WIP] Travis devel

### PR DESCRIPTION
**Pull request by LanderU**

### CODE REVIEW

**Comment by LanderU** [comment]
We need to fix all the linter errors, but the infrastructure is working.

File: aztarna/ros/ros/scanner.py [hunk 1]
```diff
@@ -44,7 +43,7 @@ def __init__(self):
             ros_host = ROSHost(address, port)
             async with self.semaphore:
                 try:
-                    code, msg, val = await ros_master_client.getSystemState('')
+                    code, val = await ros_master_client.getSystemState('')
```
Comment (XabierPB) [inline aztarna/ros/ros/scanner.py]: I think this is wrong.

File: aztarna/ros/sros/scanner.py [hunk 2]
```diff
@@ -40,7 +39,7 @@ def __init__(self):
             sros_host = None
             logger.warning('Connecting to {}:{}'.format(address, master_port))
             try:
-                master_address, port, master_cert = await get_sros_certificate(address, master_port, timeout)
+                port, master_cert = await get_sros_certificate(address, master_port, timeout)
```
Comment (XabierPB) [inline aztarna/ros/sros/scanner.py]: I think this is wrong.

File: aztarna/ros/ros/scanner.py (hunk 1 above)
Comment (LanderU) [reply aztarna/ros/ros/scanner.py]: The warning was, unused variable. But, if should be like this I'm going to revert the change.

File: aztarna/ros/sros/scanner.py (hunk 2 above)
Comment (LanderU) [reply aztarna/ros/sros/scanner.py]: The warning was, unused variable. But, if should be like this I'm going to revert the change.

File: aztarna/helpers.py [hunk 3]
```diff
@@ -53,7 +53,7 @@ class PortScanner:
         """
         conn = asyncio.open_connection(ip, port)
         try:
-            reader, writer = await asyncio.wait_for(conn, timeout=3)
```
Comment (LanderU) [inline aztarna/helpers.py]: Should I revert this change as well? @XabierPB

**Comment by LanderU** [comment]
i'm going to continue with this when ros2 is enabled.
